stage0/run.go

```diff
@@ -452,6 +457,12 @@ func Run(cfg RunConfig, dir string, dataDir string) {
 		log.Fatalf("error clearing FD_CLOEXEC on lock fd")
 	}
 
+	tpmEvent := fmt.Sprintf("rkt: Rootfs: %s Manifest: %s Stage 1 args: %s", cfg.CommonConfig.RootHash, cfg.CommonConfig.ManifestData, strings.Join(args, " "))
+	// If there's no TPM available or there's a failure for some other
+	// reason, ignore it and continue anyway. Long term we'll want policy
+	// that enforces TPM behaviour, but we don't have any infrastructure
+	// around that yet.
+	tpm.Extend(tpmEvent)
```

Comment:
Hm. This is going to be to localhost, so negative outcomes here are either going to be connection refused or something unexpected running on the port. What's the case you're concerned with?